src/loaders/genome_collection/parse_PMI_biolog_data.py

```diff
@@ -171,6 +172,7 @@ def generate_pmi_biolog_heatmap_data(
     :param kbase_collection: the name of the KBase collection
     :param root_dir: the root directory for the data to be processed
     :param upa_map_file: the path to the JSON file containing the mapping between KBase PROD UPA to other environment UPA
+    :param file_group: the file group permission for created data files
```

Comment:
If it's None what's the behavior?

Reply:
if file_group is set to None, we will skip os.chown.

Reply:
Is that a legal group name?

Reply:
Add to docstring?

Reply:
👍

Reply:
:+1: